tests/validation/tests/v3_api/test_auth.py

```diff
@@ -506,35 +527,34 @@ def get_tls(certificate):
 
 
 def enable_openldap(username, token, enable_url=CATTLE_AUTH_ENABLE_URL,
-                    password=AUTH_USER_PASSWORD, nested=False,
+                    password=OPENLDAP_AUTH_USER_PASSWORD, nested=False,
```

Comment:
why the default value here is `OPENLDAP_AUTH_USER_PASSWORD` not `PASSWORD`? 

In all other places `AUTH_USER_PASSWORD` is replaced by `PASSWORD`, and  `PASSWORD` is equal to `OPENLDAP_AUTH_USER_PASSWORD` when AUTH_PROVIDER == "openLdap".